code-challenges.test.js

```diff
@@ -13,55 +13,180 @@
+/* PSEUDOCODE
+    -create a function named objectsArraySorted
+    - input: it will take in an object
+    - process: 
+        -take in the object, and convert it into an array using object.values method
+        -once that object is an array, use the .sort method to sort it from smallest to largest
+        -use a comparison function to sort the indexes properly.
+    - output: an array with number values sorted from least to greatest.
+*/
+
+const objectsArraySorted = (object) => {
+  let objectArray = Object.values(object)
+  return objectArray.sort((value, index) => value - index)
+}
+
+// PASS  ./code-challenges.test.js
+// fibonacci
+//   ✓ takes in a number greater than 2, and returns an array that length containing the numbers of the Fibonacci sequence (3 ms)
+// objectsArraySorted
+//   ✓ takes in an object and returns an array of the values sorted from least to greatest (1 ms)
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// a) Create a test with expect statements for each of the variables provided.
+describe("summedArray", () => {
+  it("takes in an array and returns an array of the accumulating sum", () => {
+    const accountTransactions1 = [100, -17, -23, -9]
+    // Expected output: [100, 83, 60, 51]
+    const accountTransactions2 = [250, -89, 100, -96]
+    // Expected output: [250, 161, 261, 165]
+    const accountTransactions3 = []
+    // Expected output: []
+    expect(summedArray(accountTransactions1)).toEqual([100, 83, 60, 51])
+    expect(summedArray(accountTransactions2)).toEqual([250, 161, 261, 165])
+    expect(summedArray(accountTransactions3)).toEqual([])
+    
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+// fibonacci
+//   ✓ takes in a number greater than 2, and returns an array that length containing the numbers of the Fibonacci sequence (2 ms)
+// objectsArraySorted
+//   ✓ takes in an object and returns an array of the values sorted from least to greatest (1 ms)
+// summedArray
+//   ✕ takes in an array and returns an array of the accumulating sum (1 ms)
+
+// ● summedArray › takes in an array and returns an array of the accumulating sum
+
+//   ReferenceError: summedArray is not defined
 
-const accountTransactions3 = []
-// Expected output: []
 
 // b) Create the function that makes the test pass.
+/* PSEUDOCODE
+    - Create a function called summedArray
+    - input: array
+    - process: Need a variable to hold the sum of the numbers
```

Comment:
You could name the variable and also talk about the initial value you will set it to.